### PR TITLE
Added schema definition for yaml files in .neuro

### PR DESCRIPTION
**Pull request by jordyantunes**
This version includes changes to the workflow and project configuration files to enable code completion and validation by adding schema references to Apolo's config yaml files. These changes will improve the development experience by providing better code completion and validation.

Changes:

* [`{{cookiecutter.flow_dir}}/.neuro/live.yml`](diffhunk://#diff-65c776094486a5b8223092f718af818688af6b94757cf828f9647a11c7c0deb7R1-R2): Added schema reference for code completion support in the live workflow configuration file.
* [`{{cookiecutter.flow_dir}}/.neuro/project.yml`](diffhunk://#diff-2e2b83946dd5c9b293f05ec64dfafcff3258a85c7d1c1d1089a2aaab512a7a3bR1-R2): Added schema reference for code completion support in the project configuration file.

### CODE REVIEW

**Comment by YevheniiSemendiak** [comment]
Please, add changelog.

Here's an example:
https://github.com/neuro-inc/flow-template/tree/747908964c680f6679c45e1d9723d35c9e9d31af/CHANGELOG.D

For reference, we are using [towncrier ](https://github.com/twisted/towncrier) to generate changelogs.
tl; dr: put a file into the `CHANGELOG.D` folder, the filename is `<pr-number>.(feature|bugfix|misc)`, the content is a description of changes applied in a PR.

**Comment by YevheniiSemendiak** [comment]
Let's also do the same here: https://github.com/neuro-inc/flow-template-barebone

**Review by costefan — APPROVED** [review]
Looks good, thanks!
